lib/datadog/appsec/event.rb

```diff
@@ -165,6 +174,18 @@ def gzip(value)
           gz.close
           sio.string
         end
+
+        # Propagate to downstream services the information that the current distributed trace is
+        # containing at least one ASM security event.
+        def add_distributed_tags(trace)
+          return unless trace
+
+          trace.set_tag(
+            Datadog::Tracing::Metadata::Ext::Distributed::TAG_DECISION_MAKER,
+            Datadog::Tracing::Sampling::Ext::Decision::ASM
+          )
+          trace.set_tag(Datadog::AppSec::Ext::TAG_DISTRIBUTED_APPSEC_EVENT, '1')
```

Comment:
Is `Datadog::AppSec::Ext::TAG_DISTRIBUTED_APPSEC_EVENT` supposed to be a string `1` or number `1`?

Reply:
It is supposed to be a string according to [these system-tests](https://github.com/DataDog/system-tests/blob/main/tests/appsec/test_asm_standalone.py#L82)